index.html

```diff
@@ -26,14 +26,14 @@
           <a class="nav-link text-primary" href="#">Contact</a>
         </li>
       </ul>
-    </header>
-    <section class="container position-absolute start-50 top-50 translate-middle">
+    </header> 
+    <section class="container-fluid mt-5 pt-5">
       <h1 class="text-primary text-center"><dt>Hey there. I’m Alex.<br></dt>
       <span class="text-success">I'm a software developer</span></h1>
-      <p class="text-primary text-center"> I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+      <p class="text-primary text-center"> I can help you build a product , feature or website Look through some of my work and<br>experience! If you like what you see and have a project you need coded,<br> 
```

Comment:
nice work,
need to work on positioning the text, make it a little bigger 
and increase the top margin